jest/jest.config.js

```diff
@@ -26,5 +26,6 @@ module.exports = {
     'github-markdown-css': '<rootDir>/node_modules/identity-obj-proxy',
     '\\.(png)$': '<rootDir>/node_modules/identity-obj-proxy',
     '\\.(svg)$': '<rootDir>/jest/unit/empty.ts',
+    '\\.(woff|woff2)$': '<rootDir>/jest/unit/fileMock.ts',
```

Comment:
we don't need this

Reply:
I believe we need this to mock woff and woff2 files, otherwise it won’t pass in the jest test.. I have tried ‘identity-obj-procy’ but it doesn’t really works